mbed_lstools/lstools_base.py

```diff
@@ -176,6 +176,7 @@ def __init__(self):
         "9012": "SEEED_TINY_BLE",
         "9900": "NRF51_MICROBIT",
         "FFFF": "K20 BOOTLOADER",
+        "OSHW":"Unknown DAPLink Implementation",
```

Comment:
The ID's you are using to check if it is an mbed board is the daplink id, there is no difference. It comes straight from @sg- , use `UNKNOWN_DAPLINK` or similar. If you want to argue it go talk to him.